src/tests/zeroex/out_of_price.rs

```diff
@@ -228,16 +240,28 @@ async fn buy() {
                     // Way too little...
                     "sellAmount": "1",
                     "buyAmount": "1000000000000000000000",
+                    "fullSellAmount": "1",
+                    "fullBuyAmount": "100000000000000000000",
                     "kind": "buy",
                     "partiallyFillable": false,
                     "class": "market",
+                    "sellTokenSource": "erc20",
+                    "buyTokenDestination": "erc20",
+                    "preInteractions": [],
+                    "postInteractions": [],
+                    "owner": "0x5b1e2c2762667331bc91648052f646d1b0d35984",
+                    "validTo": 0,
+                    "appData": "0x6000000000000000000000000000000000000000000000000000000000000007",
```

Comment:
Any reason for this appdata specifically. The `6` and `7` doesn't look very intentional/required but also not totally random.